src/xenia/app/emulator_window.cc

```diff
@@ -174,10 +182,71 @@ bool EmulatorWindow::Initialize() {
   // FIXME: This code is really messy.
   auto main_menu = MenuItem::Create(MenuItem::Type::kNormal);
   auto file_menu = MenuItem::Create(MenuItem::Type::kPopup, L"&File");
+  // Recent Sub tap
+  auto recent_list = MenuItem::Create(MenuItem::Type::kPopup, L"&Open Recent");
+  //******Add Recent PATHS to menu***************************************
+  std::string LINES[10];
+  std::ifstream RECENT_FILE("recent.txt");
+  int i = 0;
+  // Conver from UTF-8 to whatever tf
+  std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
```

Comment:
We have helper functions for this. `xe::to_string`/`xe::to_wstring`.

Reply:
Code is using helper functions now